src/Compilers/CSharp/Portable/BoundTree/BoundNodeExtensions.cs

```diff
@@ -94,5 +97,163 @@ private class ContainsAwaitVisitor : BoundTreeWalkerWithStackGuardWithoutRecursi
                 return null;
             }
         }
+
+        public static bool VisitBinaryOperatorInterpolatedString<TArg, TInterpolatedStringType>(
```

Comment:
>public static bool VisitBinaryOperatorInterpolatedString<TArg, TInterpolatedStringType>( [](http://example.com/codeflow?start=8&length=88)

Please provide clear documentation what call backs are supposed to be invoked and in what relative order.